src/CloudModelGenerator/Program.cs

```diff
@@ -31,12 +49,14 @@ static int Main(string[] args)
                 }
 
                 const string CURRENT_DIRECTORY = ".";
-                string outputDir = outputDirOption.Value() ?? CURRENT_DIRECTORY;
+                string outputDir = passedSetOutputDir.Equals("") ? CURRENT_DIRECTORY : passedSetOutputDir;
```

Comment:
above, you can use `configuration.GetValue()` and specify a default value (e.g. null). later on, you can check for nulls which would make the code more transparent...